src/frontend/org/voltdb/importclient/SocketStreamImporter.java

```
@@ -125,6 +136,13 @@ public void run() {
                         if (!callProcedure(invocation)) {
                             System.out.println("Inserted failed: " + line);
                         }
+                        if (m_hasBackPressure) {
+                            try {
+                                Thread.sleep(100);
+                            } catch (InterruptedException ioe) {
+                                //
```

should we really swallow this exception may we should rethrow it as sensible extension to `RuntimeException`